tests/php/test-class-amp-theme-support.php

```diff
@@ -223,6 +233,33 @@ public function test_read_theme_support_and_support_args() {
 		$this->assertTrue( get_theme_support( AMP_Theme_Support::SLUG ) );
 	}
 
+	/**
+	 * Test theme_supports_reader_mode.
+	 *
+	 * @covers \AMP_Theme_Support::theme_supports_reader_mode.
+	 */
+	public function test_theme_supports_reader_mode() {
+		remove_all_filters( 'theme_root' );
```

Comment:
It looks like this [theme_root filter](https://github.com/ampproject/amp-wp/blob/0303912777b69b1857f2ab61bfcdd25d4361163a/tests/php/test-amp-style-sanitizer.php#L2241) changes the result of `get_template_directory()`, causing this test to fail. 

Maybe there's a better place to remove that filter, though.

Strangely, calling it in [AMP_Style_Sanitizer_Test::tearDown()](https://github.com/ampproject/amp-wp/blob/0303912777b69b1857f2ab61bfcdd25d4361163a/tests/php/test-amp-style-sanitizer.php#L29) didn't work.